.github/workflows/promote-rc.yml

```diff
@@ -47,3 +46,24 @@ jobs:
              { "title": "Branch", "value": "${env.GITHUB_REF}", "short": true },
              { "title": "Action URL", "value": "${env.GITHUB_SERVER_URL}/${env.GITHUB_REPOSITORY}/actions/runs/${env.GITHUB_RUN_ID}"}
             ]
+
+  update-apro-with-rc:
+    runs-on: ubuntu-latest
+    needs: promote-to-rc
+    name: update-apro-with-rc
+    env:
+      GIT_API_KEY: ${{ secrets.GH_API_KEY }}
+      APRO_REPO_TARGET: ${{ secrets.APRO_REPO_TARGET }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: "setup git config"
+        run: |
+          set -x
+          git config --global user.email "dev@datawire.io"
```

Comment:
Let's get this into a secret along with the user.name - be easier if we ever have to change to a different account.